Author: Steve Kemp <skx@debian.org>
Description: urlsnarf: zero-pad date.
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=298605
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/urlsnarf.c
+++ b/urlsnarf.c
@@ -68,7 +68,7 @@
 		 t->tm_hour - gmt.tm_hour);
 	tz = hours * 60 + t->tm_min - gmt.tm_min;
 	
-	len = strftime(tstr, sizeof(tstr), "%e/%b/%Y:%X", t);
+	len = strftime(tstr, sizeof(tstr), "%d/%b/%Y:%X", t);
 	if (len < 0 || len > sizeof(tstr) - 5)
 		return (NULL);
 	
